Upstream-Status: Inappropriate [distribution]

Configure distro version package

We need to configure a package that the system can use to determine the
system distribution version.

Hack: for now set to task-core-rpm.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur git.orig/zypp/ZConfig.cc git/zypp/ZConfig.cc
--- git.orig/zypp/ZConfig.cc	2011-03-14 16:30:07.492786134 -0500
+++ git/zypp/ZConfig.cc	2011-03-15 11:39:15.396801927 -0500
@@ -821,7 +821,7 @@
   ///////////////////////////////////////////////////////////////////
 
   std::string ZConfig::distroverpkg() const
-  { return "redhat-release"; }
+  { return "task-core-rpm"; }
 
   ///////////////////////////////////////////////////////////////////
 
